CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Latest Release: 1210.12.109.1004 (2017/05/18)
 * [azure-chef-extension #213](https://github.com/chef-partners/azure-chef-extension/pull/213) Fixed runlist conflicts with policy_name and policy_group settings in first-boot.json
+* [azure-chef-extension #212](https://github.com/chef-partners/azure-chef-extension/pull/212) Displaying the runlist when there are no entries

```

Comment:
This need not be added since it's not merged and has been replaced by https://github.com/chef-partners/azure-chef-extension/pull/213.